doc/content/verification_and_validation/val-flibe.md

```diff
@@ -0,0 +1,93 @@
+# val-flibe
+
+# Permeation experiment with FLiBe
+
+## Case Description
+
+In this experiment from [!cite](calderoni2008measurement), tritium permeation through molten FLiBe (2LiF–BeF$_2$) is measured using a stainless steel permeation cell composed of two internal volumes separated by a 2 mm thick nickel membrane. The bottom side of the membrane is exposed to a T$_2$/argon gas mixture at controlled partial pressures ranging from 1 to 1250 Pa, while the top side is in contact with molten FLiBe. Temperatures are varied between 500 °C and 700 °C (773–973 K). As tritium permeated through the membrane and dissolved in the FLiBe, it is carried away from the liquid surface by an argon sweep gas containing 5 $\%$ H$_2$.
+
+## Model Description
+
+The diffusion for tritium in the nickel membrane and FLiBe can be expressed by
+
+\begin{equation}
+\frac{\partial C_{T_2, \,  \text{Ni}}}{\partial t} = \nabla D_{\text{Ni}} \nabla C_{T_2, \,  \text{Ni}},
+\end{equation}
+and
+\begin{equation}
+\frac{\partial C_{T_2, \,  \text{FLiBe}}}{\partial t} = \nabla D_{\text{FLiBe}} \nabla C_{T_2, \,  \text{FLiBe}},
+\end{equation}
+
+where $C_{T_2, \,  \text{Ni}}$ and $C_{T_2, \,  \text{FLiBe}}$ represent the concentration fields in the nickel membrane and FLiBe respectively, $t$ is the time, and $D_{\text{Ni}}$ and $D_{\text{FLiBe}}$ denotes the diffusivity coefficients.
+
+The concentration in FLiBe is related to the partial pressure and concentration in the nickel membrane via the interface sorption law:
+
+\begin{equation}
+C_{T_2, \,  \text{FLiBe}} = K_{s, \,  \text{Ni}} P_{T_2, \,  \text{Ni}}^n = K_{s, \,  \text{Ni}} \left( C_{T_2, \,  \text{Ni}} RT \right)^n,
+\end{equation}
+
+where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature, $K_{s, \,  \text{Ni}}$ is the solubility for tritium in nickel, and $n$ is the exponent of the sorption law. For Sieverts' law, $n=0.5$.
+
+A Dirichlet boundary condition imposes equilibrium between tritium pressure at the bottom surface of the membrane and the tritium concentration in the membrane on Sieverts' law:
+
+\begin{equation}
+C_{T_2, \,  \text{Ni}} = K_{s, \,  \text{Ni}} P_{T_2}^n,
+\end{equation}
+
+where $P_{T_2}$ is the input pressure of tritium and $n$ is the exponent of the sorption law. Again for Sieverts' law, $n=0.5$.
+
+## Model Parameters
+
+| Parameter | Description | Value | Units | Reference |
```

Comment:
Add a title and label to the table. You should also reference the table in the text.